numpy/core/defchararray.py

```diff
@@ -1935,7 +1935,7 @@ class chararray(ndarray):
        `dtype` `object_`, `bytes_` or `str_`, and use the free functions
        in the `numpy.char` module for fast vectorized string operations.
 
-    Versus a regular NumPy array of type `str` or `unicode`, this
+    Versus a regular NumPy array of type `str_` or `unicode_`, this
```

Comment:
Hmm, `str_` and `unicode_` are aliases in Python 3. I think this documentation is outdated.

Reply:
"`` or `unicode_` ``" should be removed?

Reply:
chararray is [slated for removal](https://github.com/numpy/numpy/issues/23999#issuecomment-1600602108) in NumPy2.0. I think copying the text from above will suffice for now, and cross-references are optional since they are used two lines above.
```suggestion
    Versus a NumPy array of dtype `bytes_` or ``str_`, this
```